src/sagemaker/huggingface/model.py

```diff
@@ -138,12 +138,10 @@ def __init__(
                 unless ``image_uri`` is provided.
             tensorflow_version (str): TensorFlow version you want to use for
                 executing your inference code. Defaults to ``None``. Required unless
-                ``pytorch_version`` is provided. List of supported versions:
-                https://github.com/aws/sagemaker-python-sdk#huggingface-sagemaker-estimators.
+                ``pytorch_version`` is provided. The current supported version is ``2.4.1``.
             pytorch_version (str): PyTorch version you want to use for
                 executing your inference code. Defaults to ``None``. Required unless
-                ``tensorflow_version`` is provided. List of supported versions:
-                https://github.com/aws/sagemaker-python-sdk#huggingface-sagemaker-estimators.
+                ``tensorflow_version`` is provided. The current supported versions are ``1.7.1`` and ``1.6.0``.
```

Comment:
we will need to swap this. I this mixed tensorflow_version with pytorch_version

Reply:
Not sure I understand the issue, I get this reference from here

https://github.com/aws/sagemaker-python-sdk/blob/488adba580378892cfa138efd578bfc7aad2e856/src/sagemaker/huggingface/processing.py#L69-L74

Do we need to swap the version number?